dist/index.js

```diff
@@ -1,90 +1,115 @@
-'use strict';
```

Comment:
`dist/*` files should only be commited on releases, so please remove these changes from this branch.